### PR TITLE
Ak events

### PR DESCRIPTION
**Pull request by annakrystalli**
Add ReproHack and RSS events

### CODE REVIEW

File: _events/workshop-2020-05-14-remote-reprohack.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Ultimately, engaging with such materials is the only way to assess how reproducible papers are ‘out of the box’, evaluate how successful current practices are and for what purpose and identify what works and where the most pressing weaknesses in our approaches are.
+
+***
+
+Read more about previous ReproHacks on the Software Sustainability Institute blog:
+
+- Manchester Carpentry Connect ReproHack:  https://www.software.ac.uk/blog/2019-08-07-reprohacking-carpentryconnect-manchester-2019
+
+- Leiden NL ReproHack: https://www.software.ac.uk/blog/2020-01-15-reproducibility-hackathon-netherlands-aftermath
```
Comment (willfurnass) [inline _events/workshop-2020-05-14-remote-reprohack.md]:
```suggestion
Read more about previous ReproHacks on the [Software Sustainability Institute](https://www.software.ac.uk)'s blog:

- [Manchester Carpentry Connect ReproHack](https://www.software.ac.uk/blog/2019-08-07-reprohacking-carpentryconnect-manchester-2019)
- [Leiden NL ReproHack](https://www.software.ac.uk/blog/2020-01-15-reproducibility-hackathon-netherlands-aftermath)
```

File: _events/workshop-2020-05-14-remote-reprohack.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+During a ReproHack, participants attempt to reproduce published research of their choice from a list of proposed papers with publicly available associated code and data.
+
+Participants get to work with other people’s material in a low-pressure environment, record their experiences on a number of key aspects, including reproducibility, transparency and reusability of materials. and feedback to the authors.
+
+
+
+### Ways to Participate
+
+#### Submit a paper
+In the run-up to the event, you can submit details of your paper, code and data for reproduction and review here: https://sheffield-university.shinyapps.io/n8cir-reprohacks/
```
Comment (willfurnass) [inline _events/workshop-2020-05-14-remote-reprohack.md]:
```suggestion
In the run-up to the event, you can [submit details of your paper, code and data for reproduction and review here](https://sheffield-university.shinyapps.io/n8cir-reprohacks/).
```

**Review by willfurnass — CHANGES_REQUESTED** [review]
Looks good.  I've suggested two small formatting changes.

Would be good to add other Reprohacks to list of past events at some point but can leave for another PR.

**Comment by willfurnass** [comment]
NB the embedded EventBrite frame isn't showing for me on the ReproHack event page in either Firefox or Chromium (I get an `X-Frame-Options` warning placeholder in Firefox).  Don't think that should block the merging of this PR - will open a separate Issue for this.

File: _events/workshop-2020-05-14-remote-reprohack.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+category: workshop
+date: 2020-05-14
+published: true
+from: "10:00"
+to: "17:00"
+location: "Online"
+speaker: "Anna Krystalli"
+institute: "University of Sheffield"
+title: "N8 Remote ReproHack"
+eventbrite_id: https://www.eventbrite.co.uk/e/remote-reprohack-tickets-102069657092
```
Comment (willfurnass) [inline _events/workshop-2020-05-14-remote-reprohack.md]: Recommend deleting this `eventbrite_id` line as causes errors to be rendered; see Issue #244 for more info.

**Review by willfurnass — CHANGES_REQUESTED** [review]

File: _events/workshop-2020-05-14-remote-reprohack.md (hunk 3 above)
Comment (annakrystalli) [reply _events/workshop-2020-05-14-remote-reprohack.md]: Done in https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/pull/243/commits/2b3972e7381c5e68168fd2b1a5eaca699ef85cf2

**Review by willfurnass — APPROVED** [review]

**Pull request merged**